internal/sanitize_input.go

```diff
@@ -0,0 +1,8 @@
+package internal
+
+import "strings"
+
+func SanitizeInput(input string) string {
+	escaped := strings.Replace(input, "\n", "", -1)
```

Comment:
Shouldn't we replace the newline with a space at least?

Reply:
That makes sense to me, I changed it as suggested.